tests/unit-tests/server/class-llms-rest-test-students-progress-controller.php

```diff
@@ -87,6 +90,44 @@ public function test_delete_item() {
 
 	}
 
+	/**
+	 * Test delete progress of non existing post.
+	 *
+	 * @since [version]
+	 *
+	 * @return void
+	 */
+	public function test_delete_item_non_existing_post() {
+
+		$route = $this->get_route( $this->user_student, 1234567 );
```

Comment:
```suggestion
		$route = $this->get_route( $this->user_student, $this->factory->post->create() + 1 );
```

While this will work and will probably never result in an issue I've taken to incrementing a real post by one when trying to test for non-existent posts. This will safeguard against future issues if for some (unlikely) reason post ID 1234567 exists.

Reply:
I agree, I usually do this but it looked paranoid, but yeah paranoid is the new safe. <3

Reply:
fixed in #afce277